CMakeLists.txt

```diff
@@ -25,19 +98,19 @@ find_package(OpenMP)
 ###########
 ## Build ##
 ###########
-
```

Comment:
Please revert this line to keep the diff minimum.

Reply:
Added new line 36a9ad5